packages/ui/src/components/Visualization/Custom/ItemDeleteStep.tsx

```diff
@@ -18,14 +18,17 @@ export const ItemDeleteStep: FunctionComponent<ItemDeleteStepProps> = (props) =>
   const onRemoveNode = useCallback(async () => {
     if (props.loadModal) {
       /** Open delete confirm modal, get the confirmation  */
-      const isDeleteConfirmed = await deleteModalContext?.deleteConfirmation();
+      const isDeleteConfirmed = await deleteModalContext?.deleteConfirmation({
+        title: 'Permanently delete step',
+        text: 'Step parameters and its children will be lost.',
```

Comment:
This is a confusing message. Do parameters have children? Why not sticking to what I suggested?